src/panels/ActionsPanel/CharmActionsPanel/index.ts

```diff
@@ -1,2 +1,2 @@
 export { default } from "./CharmActionsPanel";
-export { Label as CharmActionsPanelLabel } from "./types";
+export { Label as CharmActionsPanelLabel } from "./ConfirmationDialog/types";
```

Comment:
This export should be in `ConfirmationDialog/index.ts` as `ConfirmationDialogLabel` and this component should export the label defined in `./types`.